chroma-manager/chroma_core/models/target.py

```diff
@@ -1584,7 +1583,9 @@ def save(self, force_insert = False, force_update = False, using = None):
         # this host
         if issubclass(self.target.downcast_class, ManagedMgs):
             from django.db.models import Q
-            other_mgs_mountables_local = ManagedTargetMount.objects.filter(~Q(id = self.id), target__in = ManagedMgs.objects.all(), host = self.host).count()
+            other_mgs_mountables_local = ManagedTargetMount.objects.filter(~Q(id=self.id),
```

Comment:
Why does "Q" need to be used here?

Reply:
don't know to be honest, don't think I changed that, would have to look into it